jOOQ-examples/jOOQ-spring-boot-example/pom.xml

```diff
@@ -11,7 +11,7 @@
 
     <groupId>org.jooq</groupId>
     <artifactId>jooq-spring-boot-example</artifactId>
-    <version>${org.jooq.version}</version>
+    <version>${jooq.version}</version>
```

Comment:
I agree with @michael-simons. If you override Boot's `${jooq.version}` then you'll get a consistent version for all of jOOQ as we manage the version of its main modules and its Maven plugin all with the same `jooq.version` property (as long as you're using `spring-boot-starter-parent` which you are here).